.github/workflows/initiate_release.yml

```diff
@@ -0,0 +1,47 @@
+name: Create release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The new version number with 'v' prefix. Example: v1.40.1"
+        required: true
+
+jobs:
+  init_release:
+    name: 🚀 Create release PR
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # gives the changelog generator access to all previous commits
+
+      - name: Update CHANGELOG.md, package.json and push release branch
+        env:
+          VERSION: ${{ github.event.inputs.version }}
+        run: |
+          npx --yes standard-version@9.3.2 --release-as "$VERSION" --skip.tag --skip.commit --tag-prefix=v
```

Comment:
better to add `standard-version` to devDeps

Reply:
✅